lms/templates/certificates/certificate-base.html

```diff
@@ -0,0 +1,35 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! import mako.runtime %>
+<% mako.runtime.UNDEFINED = '' %>
+
+<%
+    # set doc language direction
+    from django.utils.translation import get_language_bidi
+    dir_rtl = 'rtl' if get_language_bidi() else 'ltr'
+%>
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head dir=-"${dir_rtl}">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>${document_title}</title>
+
+        <!-- TODO: add description -->
+        <meta name="description" content="">
+
+        <!-- TODO: add google analytics -->
```

Comment:
Hmm, I'm not sure, but ideally it'd be the same parameters so that end instance owners see all analytics in one place.

@marcotuts, any suggestions or requirements for this now?